Tests/Tracking/APITest.swift

```diff
@@ -107,4 +108,120 @@ class TrackingAPITest: UnitTest {
             config.autoScreenViewBody = { [:] }
         }
     }
+
+    // SDK wrappers can configure the SDK from a Map.
+    // This test is in API tests as the String keys of the Map are public and need to not break for the SDK wrappers.
+    func test_createSdkConfigFromMap() {
+        let trackingApiUrl = String.random
+        let autoTrackPushEvents = false
+        let backgroundQueueMinNumberOfTasks = 10000
+        let backgroundQueueSecondsDelay: TimeInterval = 100000
+        let backgroundQueueExpiredSeconds: TimeInterval = 100000
+        let logLevel = "info"
+        let autoTrackScreenViews = true
+        let autoTrackDeviceAttributes = false
+        let sdkWrapperSource = "Flutter"
+        let sdkWrapperVersion = "1000.33333.4444"
+
+        let givenParamsFromSdkWrapper: [String: Any] = [
+            CioSdkConfig.Keys.trackingApiUrl.rawValue: trackingApiUrl,
+            CioSdkConfig.Keys.autoTrackPushEvents.rawValue: false,
+            CioSdkConfig.Keys.backgroundQueueMinNumberOfTasks.rawValue: backgroundQueueMinNumberOfTasks,
+            CioSdkConfig.Keys.backgroundQueueSecondsDelay.rawValue: backgroundQueueSecondsDelay,
+            CioSdkConfig.Keys.backgroundQueueExpiredSeconds.rawValue: backgroundQueueExpiredSeconds,
+            CioSdkConfig.Keys.logLevel.rawValue: logLevel,
+            CioSdkConfig.Keys.autoTrackScreenViews.rawValue: autoTrackScreenViews,
+            CioSdkConfig.Keys.autoTrackDeviceAttributes.rawValue: autoTrackDeviceAttributes,
+            CioSdkConfig.Keys.source.rawValue: sdkWrapperSource,
+            CioSdkConfig.Keys.sourceVersion.rawValue: sdkWrapperVersion
```

Comment:
We will need to go back to using hard-coded strings in this test function. 

If this was a unit test, using constants would be nice. But because this is an API test, we should be using hard-coded strings. 

Why? 

If someone was to, for example, modify `CioSdkConfig.Keys.trackingApiUrl` to `CioSdkConfig.Keys.trackingUrl`. If this test function used constants, tests would pass. But if it was hard-coded, this test would fail. 

Since this test function's purpose is to make sure that we don't, by accident, introduce breaking changes to the public API of our SDK, we would want the test to fail reminding us that we might have a Flutter developer out there using `"trackingApiUrl"` in their app.